test/Mono.Linker.Tests.Cases/Inheritance.AbstractClasses/NoKeptCtor/OverrideRemoval/Dependencies/OverrideOfAbstractIsKeptNonEmptyLibraryWithNonEmpty.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mono.Linker.Tests.Cases.Inheritance.AbstractClasses.NoKeptCtor.OverrideRemoval.Dependencies
+{
+	public class OverrideOfAbstractIsKeptNonEmptyLibraryWithNonEmpty :
```

Comment:
nit: I'm not sure what the "non-empty" in `NonEmptyLibrary` or `WithNonEmpty` refer to - maybe the names could be simplified?

Reply:
The idea was to keep the names starting with the name of the test - so that it's easy to tell which dependencies belong to which test.